packages/lesswrong/components/users/DialogueSuggestionsPage.tsx

```diff
@@ -0,0 +1,782 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { useTracking } from "../../lib/analyticsEvents";
+import { gql, useQuery, useMutation } from "@apollo/client";
+import { useUpdateCurrentUser } from "../hooks/useUpdateCurrentUser";
+import { useCurrentUser } from '../common/withUser';
+import { randomId } from '../../lib/random';
+import { commentBodyStyles } from '../../themes/stylePiping';
+import { useCreate } from '../../lib/crud/withCreate';
+import { useNavigation } from '../../lib/routeUtil';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { useSingle } from '../../lib/crud/withSingle';
+import { useMulti } from "../../lib/crud/withMulti";
+import ReactConfetti from 'react-confetti';
+
+const { NewConversationButton, UsersName, PostsTooltip, LoadMore } = Components;
```

Comment:
Conventionally, we extract these inside the component, though I don't think there's really anything that requires that

Reply:
I made this change, though since we use different ones in different components that meant having three separate lines of code - but that's fine I suppose